openmetrics/datadog_checks/openmetrics/data/conf.yaml.example

```diff
@@ -171,6 +171,11 @@ instances:
     #
     # exclude_metrics_by_labels: {}
 
+    ## @param tag_by_endpoint - boolean - optional - default: true
+    ## Whether to include an endpoint tag or not.
+    #
+    tag_by_endpoint: false
```

Comment:
Small nit, should the default value of the parameter match the ootb value of this parameter?